.packit.yaml

```diff
@@ -1,11 +1,5 @@
 # See the documentation for more information:
 # https://packit.dev/docs/configuration/
-actions:
-    changelog-entry:
-        - bash -c 'echo "- New upstream release"'
-    post-upstream-clone:
```

Comment:
we will need this chunk, as our spec file is technically stored down stream, and since this service is running here, we need to get a copy of it.